qa/suites/upgrade/quincy-p2p/quincy-p2p-stress-split/4-workload/rbd_api.yaml

```diff
@@ -0,0 +1,10 @@
+meta:
+- desc: |
+   librbd C and C++ api tests
+stress-tasks:
+- workunit:
+     branch: quincy
+     clients:
+        client.0:
+           - rbd/test_librbd.sh
```

Comment:
This workunit needs at least `is full \(reached quota` and `\(POOL_FULL\)` and likely more in the `log-ignorelist`.

Reply:
Thanks @idryomov. See my question above-- just to understand the difference here compared to Pacific.